crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -114,6 +117,366 @@ pub enum DbValidationError {
     LastBefore,
     #[error("Pagination is currently disabled on balances")]
     PaginationDisabledOnBalances,
+    #[error(
+        "Cannot provide owner address for an owner that is not of owner type Address or Object"
```

Comment:
this is confusing. Why not just:

```suggestion
        "Invalid owner type. Must be Address or Object"
```